src/main/java/com/gmail/val59000mc/commands/HealCommandExecutor.java

```diff
@@ -0,0 +1,48 @@
+package com.gmail.val59000mc.commands;
+
+import com.gmail.val59000mc.exceptions.UhcPlayerNotOnlineException;
+import com.gmail.val59000mc.game.GameManager;
+import com.gmail.val59000mc.languages.Lang;
+import com.gmail.val59000mc.players.PlayerManager;
+import com.gmail.val59000mc.players.UhcPlayer;
+import org.bukkit.Bukkit;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+public class HealCommandExecutor implements CommandExecutor {
+    private final PlayerManager playerManager;
+    private final GameManager gameManager;
+
+    public HealCommandExecutor(PlayerManager playerManager, GameManager gameManager) {
+        this.playerManager = playerManager;
+        this.gameManager = gameManager;
+    }
+
+    @Override
+    public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
+        if (sender.hasPermission("uhc-core.commands.heal")) {
+            if (args.length == 0) { // heal all players
```

Comment:
I prefer healing the sender instead

Reply:
This repo is no longer maintained here, please submit it on https://gitlab.com/uhccore/uhccore